### PR TITLE
Makes comment section that allows for comments to be posted

### PR DESCRIPTION
**Pull request by cjoshea9**
Adds the comment section at the bottom of the home page. The comments made will stay on the page after a refresh.

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -42,14 +42,15 @@ function showSlide(n) {
   slides[slideIndex].style.display = "block";
 }
 
-async function getVacationSpots() {
+async function getComments() {
```
Comment (g-harel) [inline portfolio/src/main/webapp/script.js:45]: It's not being called more than once from what I can tell, but clearing the contents of `data-servlet` before filling it again would be a good precaution.

Comment (cjoshea9) [reply portfolio/src/main/webapp/script.js:45]: I was attempting to do that with line 50. Is there a better way to do that? Would I clear .innerText? @g-harel

Comment (g-harel) [reply portfolio/src/main/webapp/script.js:45]: Whoops, missed that. Disregard previous comment!

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -6,7 +6,7 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+  <body onload=getComments()>
```
Comment (g-harel) [inline portfolio/src/main/webapp/index.html:9]: Think this may need some quotes

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -42,14 +42,15 @@ function showSlide(n) {
...
   const response = await fetch('/data');
-  const places = await response.json();
-  const vacationList = document.getElementById('data-servlet');
-  vacationList.innerHTML = "<h1>Vacation Spots</h1>";
-  for(let i = 0; i<places.length; i++) {
+  const comments = await response.json();
+  console.log(comments);
```
Comment (g-harel) [inline portfolio/src/main/webapp/script.js:48]: Nit: remove debugging logs

**Pull request merged**